rts/Game/Camera/OverheadController.cpp

```diff
@@ -158,7 +161,7 @@ void COverheadController::MouseWheelMove(float move, const float3& newDir)
 			if ((wantedPos.y + (dir.y * newHeight)) < 0.0f)
 				newHeight = -wantedPos.y / yDirClamp;
 
-			if (newHeight < maxHeight) {
+			if (newHeight < maxHeight && newHeight > minHeight) {
```

Comment:
See like 146, there's some kind of clamping there. We might want to use minHeight instead of harcoded 60.0

Reply:
On second thought this is relevant but perhaps out of scope for this PR, I created a separate issue: https://github.com/beyond-all-reason/spring/issues/1578